src/modules/func.js

```diff
@@ -0,0 +1,83 @@
+import removeFor from './delete.js';
+import edit from './editable.js';
+import isCompleted from './completed.js';
+
+export default class UI {
+  static getlist () {
+    let tasks;
+    if (localStorage.getItem('items') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('items'));
+    }
+    return tasks;
+  }
+
+  static addtodo  (element)  {
+    const todos = this.getlist();
+    todos.push(element);
+    localStorage.setItem('items', JSON.stringify(todos));
+  }
+
+  static displaylist () {
+    const todos = this.getlist();
+    todos.forEach((element) => this.listlibrary(element));
+  }
+
+  static listlibrary (element)  {
+    const listcontainer = document.querySelector('.list-container');
+
+    const list = document.createElement('div');
+    list.classList.add('mylist');
+    const check = document.createElement('input');
+    check.type = 'checkbox';
+    // check.checked = true;
+    // check.id = 'checkbox';
```

Comment:
- [ ] Please remove the following commented codes. It is among the best practices to remove unnecessary commented codes to keep your codebase clean 😉 👍

Reply:
I will do that. Thank you